camel-idea-plugin/src/main/java/org/apache/camel/idea/annotator/CamelSimpleAnnotator.java

```diff
@@ -49,7 +49,8 @@ boolean isEnabled() {
      */
     void validateText(@NotNull PsiElement element, @NotNull AnnotationHolder holder, @NotNull String text) {
         boolean hasSimple = text.contains("${") || text.contains("$simple{");
-        if (hasSimple && CamelIdeaUtils.isCamelSimpleExpression(element)) {
+        if (hasSimple
+            && CamelIdeaUtils.isCamelSimpleExpression(element)) {
```

Comment:
This should be OR. Otherwise it would not work with 
if (hasSimple
             || CamelIdeaUtils.isCamelSimpleExpression(element)) {

    .filter().simple("xxx")

You genererally have 2 ways in java dsl, either as parameter

    .filter ( simple("xxx") )

Or as chained method call

    .filter . simple("xxx")